first draft of system-wide requirements

don-ny opened the pull request:
- drafted first version of system-wide requirements
- *Organisatorische Randbedingungen* and *Rechtliche Anforderungen* will be discussed during next meeting with PO

henrikalbrecht reviewed (changes requested):
- The Usability is heavily focusing on the administration use case. Therefore it isn`t a systemwide requirement but a requirement fpr the use case.
- If there are no performance requirements i wouldn`t ention them and just dont mention it at all. -> In the lecture we called this "shopping cart mentality"

don-ny commented:
> * The Usability is heavily focusing on the administration use case. Therefore it isn`t a systemwide requirement but a requirement fpr the use case.
> 
>     * If there are no performance requirements i wouldn`t ention them and just dont mention it at all. -> In the lecture we called this "shopping cart mentality"

- changed usability in accordance with guidelines given by PO
- deleted performance
